Makefile

```diff
@@ -36,6 +37,14 @@ else ifeq ($(AUTO_TEST), boot)
 CARGO_OSDK_ARGS += --init-args="/regression/boot_hello.sh"
 endif
 
+ifeq ($(BENCHMARK), sysbench)
+CARGO_OSDK_ARGS += --init-args="/benchmark/sysbench.sh"
+else ifeq ($(BENCHMARK), iperf3)
+CARGO_OSDK_ARGS += --init-args="/benchmark/iperf3.sh"
```

Comment:
Add this parameter after the iperf benchmark CI has actually been implemented. Currently it's useless and we don't even have `iperf3.sh`.

Reply:
Thank you for reminding! I will remove it. Actually we have iperf3 toolkit inside Asterinas, but it isn't supported as #866. We can add it after the problem is solved.